### ISSUE TITLE
Random password change - changepw.ChangePasswordServlet, 4006 

### ISSUE THREAD

**Issue by pxeboot-123** [pwm-project/pwm#407]
I happen to have a random behaviour regarding the changepassword module. 

Randomly, on some users I get the message below. Please note that I do not have any password policies on my 2012 AD. Any help would be very appreciated. 

Best,

E.

2018-08-10T09:28:34Z, DEBUG, changepw.ChangePasswordServlet, 4006 PASSWORD_BADPASSWORD (error setting password for user ... com.novell.ldapchai.exception.ChaiPasswordPolicyException: 
javax.naming.directory.InvalidAttributeValueException: [LDAP: error code 19 - 00000005: AtrErr: DSID-03191083, #1:
	0: 00000005: DSID-03191083, problem 1005 (CONSTRAINT_ATT_TYPE), data 0, Att 9005a (unicodePwd)

**Comment by bigolbear** [comment]
At my employer, we use PWM to allow self-service password management for a number of LDAP and AD directories. Every time I have seen this error it was because PWM was trying to write an attribute to the directory that did not exist. In a couple of cases, we had configured PWM to update login history in AD but the attribute did not exist and the proxy user could not create. HTH

**Comment by baughj** [comment]
I was pulling my hair out about this for a few days. The google group has a lot of people having the same problem, no solutions or responses to be found there, however. I am using pwm with Server 2019.

The _only_ workaround I discovered was the following:

1) I ended up having to give the proxy user full control to subobjects in `CN=Users` (boo)
2) I had to execute `net accounts /MINPWAGE:0` on a domain controller (as an admin).

This doesn't make much sense to me because I tested with users whose passwords were older than 1 day, but changing the minimum age to 0 is the ONLY way I could get pwm to work.
